src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -380,10 +385,10 @@ def _add_endpoint_args(parser):
     endpoint_group.add_argument(
         "--backend",
         type=str,
-        choices=utils.get_enum_names(OutputFormat)[2:],
+        choices=["tensorrtllm", "vllm"],
         default="tensorrtllm",
         required=False,
-        help=f'When using the "triton" service-kind, '
+        help=f'When using the "kserve" endpoint type, '
```

Comment:
Can generate endpoint not use trtllm vs vllm?

Reply:
It can - I haven't added any different behavior for the different backends. Actually - it has only been tested against vllm at the moment. So this is fair point ...

Let me move this back to draft -